src/routes/account/index.svelte

```diff
@@ -0,0 +1,28 @@
+<script context="module" lang="ts">
+	import type { Load } from '@sveltejs/kit';
+	export const load: Load = ({ session, page }) => {
+		const { user, authenticated } = session;
+
+		if (!authenticated) {
+			const query = new URLSearchParams();
+			query.append('page', page.path);
+
+			return {
+				status: 302,
```

Comment:
302 is tijdelijke redirect, right? 301 is permanent.

Reply:
302 is found

Reply:
ik had liever 401 gedaan, maar bij een redirect moet je een 3xx code doen

Reply:
Ah oke.

`This response code means that the URI of requested resource has been changed temporarily. Further changes in the URI might be made in the future. Therefore, this same URI should be used by the client in future requests.`

Reply:
ik zie nu dat `302` ooit inderdaad `Temporary Redirect` was maar veranderd is naar `found`. `307` is nu `Temporary Redirect` 

Mooi weetje weer  hahaha

Reply:
Uitgecomment voor andere ticket?